### PR TITLE
csmdb_python_dependency

### PR DESCRIPTION
**Pull request by williammorrison2**
# Purpose
Python dependency (add appropriate dependencies to the CSM rpms so yum can automatically install all of the right packages.)

- [x]  @pdlun92 to review (regression)
- [x] @besawn review
- [ ] @mew2057 review
- [ ] @fpizzano  review

### CODE REVIEW

File: csmd/setupRPM.cmake [hunk 1]
```diff
@@ -30,6 +30,7 @@ set( CPACK_RPM_csm-core_POST_INSTALL_SCRIPT_FILE
 # ibm-csm-db rpm settings
 set(CPACK_RPM_csm-db_PACKAGE_ARCHITECTURE "noarch")
 set(CPACK_RPM_csm-db_PACKAGE_REQUIRES "pv")
+set(CPACK_RPM_csm-db_PACKAGE_REQUIRES "python-psycopg2")
```
Comment (tgooding) [inline csmd/setupRPM.cmake]: Did you intend to replace the "pv" package dependency with "python-psycopg2"?  Or should it be "pv,python-psycopg2"?

Comment (williammorrison2) [reply csmd/setupRPM.cmake]: @tgooding we would be including both pv and python-psycopg2.

Comment (tgooding) [reply csmd/setupRPM.cmake]: ok, looks like the second set() is dropping the pv dependency then.

Comment (williammorrison2) [reply csmd/setupRPM.cmake]: @tgooding can you combine the two? Or is there an alternative solution that you would suggest?

**Comment by pdlun92** [comment]
Passes regression and fixes the issue:
rpm dependencies for current daily master build (without fix):
```
# rpm -qpR  /ugadmin/CORAL/1.1.0/CORAL_Product/production/latest_images_ppc64le_rh75/ibm-csm-db-1.6.0-2219.noarch.rpm
pv
/bin/sh
/bin/sh
/bin/sh
/bin/sh
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(CompressedFileNames) <= 3.0.4-1
/bin/bash
/bin//bash
/bin/python
rpmlib(PayloadIsXz) <= 5.2-1
```
rpm dependencies for build from this pull-request
```
# rpm -qR ibm-csm-db-1.6.0-2222.noarch
pv
python-psycopg2
/bin/sh
/bin/sh
/bin/sh
/bin/sh
rpmlib(FileDigests) <= 4.6.0-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(CompressedFileNames) <= 3.0.4-1
/bin/bash
/bin//bash
/bin/python
rpmlib(PayloadIsXz) <= 5.2-1
```

**Pull request merged**